csharp/test/Microsoft.ML.OnnxRuntime.EndToEndTests.Capi/Microsoft.ML.OnnxRuntime.Gpu.EndToEndTests.RunCapi.vcxproj

```diff
@@ -26,13 +24,13 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'" Label="Configuration">
     <ConfigurationType>Application</ConfigurationType>
     <UseDebugLibraries>true</UseDebugLibraries>
-    <PlatformToolset>v141</PlatformToolset>
+    <PlatformToolset>v142</PlatformToolset>
```

Comment:
Why the change to toolset v142? It is a requirement now?

Reply:
The build complains Windows SDK 8.1 is not found.  The new build machine doesn't have Windows SDK for 8.1.

Reply:
Ok -- users will require same toolset to run the test projects, but that should be fine.